desk/app/groups.hoon

```diff
@@ -682,13 +692,11 @@
   ::  v8 -> v9: leave all /contact subscriptions
   ::
   =?  cor  ?=(%8 -.old)
-    %+  roll  ~(tap by wex.bowl)
-    |=  [[[=wire =dock] *] =_cor]
-    ?.  ?=([%contact ~] wire)  cor
+    =/  =dock  [our.bowl %contacts]
     =^  caz=(list card)  subs.cor
-      (~(unsubscribe s [subs bowl]) wire dock)
+      (~(unsubscribe s [subs bowl]) /contact dock)
     =.  cor  (emil:cor caz)
-    (emit:cor [%pass wire %agent dock %leave ~])
+    (emit:cor [%pass /contact %agent dock %leave ~])
```

Comment:
Actually, also confused (and I should've noticed earlier) about why we _both_ call `+unsubscribe` _and_ issue a `%leave` manually. iiuc `+unsubscribe` already issues a leave unconditionally, so you can just produce the cards from that and call it a day.